src/sass/_demos.scss

```
@@ -164,3 +164,23 @@
     }
   }
 }
+
+.demo-sidebar {
+  --gap: 3rem;
+
+  main {
+    display: grid;
+    grid-gap: var(--gap);
+    grid-template-columns: minmax(min(50vw, 30ch), 1fr) fit-content(20ch);
+  }
+
+  aside {
+    border-left: 1px solid lightslategray;
+    padding-left: var(--gap);
+
+    figure {
+      position: sticky;
+      top: var(--gap);
+    }
+  }
+}
```

Updates to prevent responsive overflow and accomdate snippet updates:

```suggestion
.demo-sidebar {
  --gap: 1rem;

  .container {
    display: grid;
    grid-gap: var(--gap);
    grid-template-columns: minmax(min(50vw, 25ch), 1fr) fit-content(20ch);
    width: min(100%, 60ch);
    margin: 2rem auto;
  }

  aside {
    border-left: 1px solid lightslategray;
    padding-left: var(--gap);
  }
}
```